docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -124,6 +144,51 @@ spec:
         modes: ['moderator', 'observer']
 ```
 
+#### Joining sessions example
+
+Here is an example of Jeff with role `prod-access` connecting to
+a SSH server in the production environment.
+
+```code
+$ tsh ssh ubuntu@prod.teleport.example.com
+Teleport > Creating session with ID: 46e2af03-62d6-4e07-a886-43fe741ca044...
+Teleport > Controls
+  - CTRL-C: Leave the session
+  - t: Forcefully terminate the session (moderators only)
+Teleport > User jeff joined the session.
+Teleport > Waiting for required participants...
+```
+
+Jeff's session is paused, waiting for the required observers.
+
+Now Alice with the `auditor` role joins as a moderator and
+the session can begin. 
+
+```code
+$ tsh join --mode=moderator 46e2af03-62d6-4e07-a886-43fe741ca044
+Teleport > Creating session with ID: 46e2af03-62d6-4e07-a886-43fe741ca044...
+Teleport > Controls
+  - CTRL-C: Leave the session
+  - t: Forcefully terminate the session (moderators only)
+Teleport > User jeff joined the session.
+Teleport > Waiting for required participants...
+Teleport > User alice joined the session.
+Teleport > Connecting to prod.teleport.example.com over SSH
+
+ubuntu@prod.teleport.example.com % 
+```
+
+Here is an example of joining from the UI that is available for Server sessions.
+
+![Join Server Session from UI](../../../img/access-controls/joinsession.png)
```

Comment:
We're about to change this, so I would either remove this section if you want to merge this now, or wait 1-2 weeks for an updated screenshot.

Reply:
Thanks let's wait then